### ISSUE TITLE
[BUG] UCI BusinessProcessFlow BooleanItem SetValue fails

### ISSUE THREAD

**Issue by jtemple2** [Microsoft/EasyRepro#869]
**EasyRepro Version**
  - [ ] Microsoft Dynamics 365 Online Version 8.2 (8.2.x) (DB 8.2.x) online
  - [ ] Microsoft Dynamics 365 Online Version 9.0 (9.0.x) (DB 9.0.x) online
  - [ ] Microsoft Dynamics 365 Online Version 9.0.2 (9.0.2) (DB 9.0.2) online
  - [x] Microsoft Dynamics 365 Online Version 9.1 (9.1.x) (DB 9.1.x) online
  
**UCI or Classic Web**
  - [x] UCI (XrmApp)
  - [ ] Classic Web (XrmBrowser)

**Online or On Premise**
  - [x] Online
  - [ ] On Premise

**Browser**
  - [x] Chrome
  - [ ] Firefox
  - [ ] IE
  - [ ] Edge

**Describe the bug**  
Cannot update Two Option field on the Business Procees Flow of a Case. 
The code fails and the error "System.ArgumentException: MoveToElement cannot move to a null element with no offset. Parameter name: toElement" is displayed.

**Special formatting / display**  
Unknown

**Code to reproduce**  
This is a custom Two Option field on our Case BPF.
This is the method we are using:
xrmApp.BusinessProcessFlow.SetValue(new BooleanItem { Name = "NAME_OF_FIELD", Value = true });
 
**Expected behavior**  
Should be able to update the field

**Screenshots**  

**Additional context**  
This is the element from the Dev Tools:
<label id="header_process_NAME_OF_FIELD-header_process_cl_signoffsupportrequired6-header_process_NAME_OF_FIELD.fieldControl-checkbox-inner-first" aria-hidden="true" data-id="header_process_NAME_OF_FIELD.fieldControl-checkbox-inner-first" class="zf zg zh ">No</label>

**Comment by TYLEROL** [comment]
@jtemple2 - If you have not, could you try testing this with the latest develop branch?

We're planning to push a new release very soon -- if you still have problems I may be able to try and get a change in if necessary.

Warm Regards,
Tyler

**Comment by TYLEROL** [comment]
@jtemple2 - We moved ahead with merging to releases/v9.1 and publishing the 9.1.0.17138 NuGet package. Please test accordingly and let us know if this problem persists.

Warm Regards,
Tyler

**Comment by jtemple2** [comment]
@TYLEROL 
Thank you for your update.

I got the latest off releases/v9,1 and modified one of the sample tests to set the value for the field on the BPF I am trying to interact with.
The test is still throwing an error though this time it's "System.NullReferenceException: 'Object reference not set to an instance of an object.'"

I checked the AppElementReferences for the xpath:
{ "BPF_BooleanFieldContainer", ".//input[contains(@data-id, \'header_process_[NAME].fieldControl-checkbox-toggle\')]" }
but that doesn't match the one for the field  I am trying to set.
I can be find the field with this xpath:
.//label[contains(@data-id, 'header_process_[NAME].fieldControl-checkbox-inner-first')]

Thank you for your help with this
Regards,
Jean
